core/src/main/java/org/apache/carbondata/core/indexstore/Blocklet.java

```diff
@@ -31,7 +31,9 @@
  */
 public class Blocklet implements Serializable {
 
-  private Path path;
+  private static final long serialVersionUID = 7483063116325388460L;
+
+  private String path;
```

Comment:
Why changed Path to string?
But constructor not changed to handle the datatype change